lib/pbench/test/unit/server/conftest.py

```diff
@@ -398,14 +363,14 @@ def attach_dataset(create_drb_user, create_user):
     # for one Dataset and letting it default for the other.
     with freeze_time("1970-01-01 00:42:00"):
         Dataset(
-            owner_id=str(create_drb_user.id),
+            owner=str(create_drb_user.username),
```

Comment:
This is a bit weird. Yeah, the validator currently allows `User` or `username` string, but I don't think the real code ever uses a string, and I'd prefer to use the `User` object. First, it's more efficient since it avoids a query to find what we already have, and second it's more reliable. (In fact, why do we even *have* that lever? 😦 It probably "seemed like a good idea at the time", but that was probably wrong...)